refactor!: remove outdated widget properties and document used widget properties (#118)
flekschas/jupyter-scatter: docs/api.md

### Widget

The widget has the following properties, which you can think of as the view model of Jupyter Scatter.

::: warning

> While you can adjust these properties directly, we highly recommend using the [`Scatter` methods](#methods) as these methods purely exist to simplify and automate the setting of the widget properties.

Reviewer: ```suggestion
While you can adjust these properties directly, the [`Scatter` methods](#methods) are the idiomatic and recommended way to set widget properties.
```
